tests/Functional/ContentOrganisationRenderTest.php

```diff
@@ -187,6 +159,129 @@ protected function assertContentHeader(NodeElement $element, string $title, stri
     $this->assertEquals($id, $header->getAttribute('id'));
   }
 
+  /**
+   * Creates Contact entity.
+   *
+   * @param string $name
+   *   Entity name. Is used as a parameter for test data.
+   * @param string $bundle
+   *   Entity bundle.
+   * @param int $status
+   *   Entity status.
+   *
+   * @return \Drupal\oe_content_entity_contact\Entity\ContactInterface
+   *   Contact entity.
+   */
+  protected function createContactEntity(string $name, string $bundle, int $status): ContactInterface {
```

Comment:
Let's create a ContentCreationTrait where we move the following functions:

- getStorage()
- createMediaImage()
- createContactEntity()
- createStakeholderOrganisationEntity()

Make use of the new trait in project, and contact tests.

Reply:
getStorage() - we don't need it.
createMediaImage() - already exist in oe_media
createContactEntity() - need to be created
createStakeholderOrganisationEntity() -> createOrganisationEntity and pass bundle as a parameter.

Reply:
As discussed with @drishu we will do this in a separate ticket, since we need to refactor tests anyway.